api/src/main/java/org/openmrs/module/ugandaemrreports/definition/cohort/evaluator/CurrentlyInProgramCohortEvaluator.java

```diff
@@ -0,0 +1,57 @@
+package org.openmrs.module.ugandaemrreports.definition.cohort.evaluator;
+
+import org.openmrs.Cohort;
+import org.openmrs.PatientProgram;
+import org.openmrs.annotation.Handler;
+import org.openmrs.module.reporting.cohort.EvaluatedCohort;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.InProgramCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.evaluator.CohortDefinitionEvaluator;
+import org.openmrs.module.reporting.evaluation.EvaluationContext;
+import org.openmrs.module.reporting.evaluation.querybuilder.HqlQueryBuilder;
+import org.openmrs.module.reporting.evaluation.service.EvaluationService;
+import org.openmrs.module.ugandaemrreports.definition.cohort.definition.CurrentlyInProgramCohortDefinition;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Date;
+import java.util.List;
+
+@Handler(supports={CurrentlyInProgramCohortDefinition.class},order = 1)
```

Comment:
@dbaluku Can you add a class comment on how this is copied from another Cohort Definition and what has changed

Reply:
Done